repos/iiif-storage/Module.php

```diff
@@ -94,6 +94,38 @@ public function attachListeners(SharedEventManagerInterface $sharedEventManager)
                             ])
                         ->setValue($form->getSiteSettings()->get('original-ids', false))
                     )
+                    ->add(
+                        (new Text('collections-per-page'))
```

Comment:
We should validate that the input coming in is a valid number (or you might end up casting the result to 0 and get 0 collections per page).